crates/gosub_styling/Cargo.toml

```diff
@@ -15,9 +15,19 @@ lazy_static = "1.5"
 anyhow = "1.0.86"
 regex = "1.10.5"
 colors-transform = "0.2.11"
+vello = "0.1.0"
```

Comment:
I think vello can be removed here. But I guess that's also happening when we resolve the current conflicts, so just a side note to check if it really is gone afterwards